t/translator.t

```diff
@@ -4,8 +4,17 @@ use v5.16;
 use warnings;
 use utf8;
 
+use POSIX qw (setlocale);
+use Locale::Messages qw[LC_MESSAGES LC_ALL LC_CTYPE];
 use Test::More;
 
+# Set correct locale for translation in case not set in calling environment
+delete $ENV{"LANG"};
+delete $ENV{"LANGUAGE"};
+setlocale( LC_CTYPE, "C.UTF-8");
+setlocale( LC_ALL, "C.UTF-8");
+setlocale( LC_MESSAGES, "C.UTF-8");
```

Comment:
Isn't it enough to just set LC_ALL? That one should override the others.

Reply:
> Isn't it enough to just set LC_ALL? That one should override the others.

Unfortunately, there is more to it.